scipy/special/_precompute/utils.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import division, print_function, absolute_import
+import os
+import numpy as np
+from numpy.testing import dec, assert_
+
+try:
+    import mpmath as mp
+except ImportError:
+    try:
+        import sympy.mpmath as mp
+    except ImportError:
+        pass
+
+try:
+    from sympy.abc import x
+except ImportError:
+    pass
+
+
+def skip():
+    msg = "Set environment variable SCIPY_PRECOMPUTE=1 to run precompute tests."
```

Comment:
You can maybe skip the tests if sympy isn't installed, cf. `test_mpmath.py:mpmath_check`